CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ All notable changes to [diagram-js-grid](https://github.com/bpmn-io/diagram-js-g
 
 ___Note:__ Yet to be released changes appear here._
 
+## 1.1.0
+
+* `DEPS`: update to `diagram-js@15.1.0`
```

Comment:
This is not `DEPS`, as this is a test dependency only.

I'd hence frame this as

```
* `FEAT`: support `diagram-js@15`
```